components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/internal/cache/ApplicationResourceIDCacheInboundAuthKey.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
This should be wso2 opensource license. Fix in all places

Reply:
fixed